optimum/habana/transformers/models/gptj/modeling_gptj.py

```diff
@@ -390,7 +390,7 @@ def gaudi_gptj_model_forward(
             def create_custom_forward(module):
                 def custom_forward(*inputs):
                     # None for past_key_value
-                    return module(*inputs, use_cache, output_attentions)
+                    return module(*inputs, use_cache, output_attentions, token_idx, sin, cos)
```

Comment:
`token_idx` should be False when using gradient checkpointing right?

Reply:
It's `None` actually

Reply:
fixed to `None`